bnd-workspace/de.fraunhofer.abm.app/web/controllers/CollectionController.js

```diff
@@ -77,7 +77,7 @@ function collectionController($rootScope, $scope, $http, $location, $route, $rou
 				if(d.status == '403') {
 					$location.path('/login');
 				} else {
-					Notification.error('Failed with ['+ d.status + '] '+ d.statusText);
+					Notification.error('Collections cannot be retrieved due to some internal error');
```

Comment:
Please reformulate: "Internal error: the collections cannot be retrieved. Please try again later. If the error persists, please report it here: https://github.com/ABenchM/abm/issues"
Also reformulate in other places.